.github/workflows/run_periodic_tests.yml

```diff
@@ -0,0 +1,128 @@
+# Run all unit tests and integration tests for all Python versions 
+# and platforms at 3am UTC every day and on PRs to the main branch
+name: PyBaMM
```

Comment:
Maybe something like -
```suggestion
name: Scheduled tests
```

Reply:
I agree this would be more informative, but I think it might also change the name in the badge we have in the README. Not sure if there is a way to have another name in the badge.

Reply:
Maybe we can rename it to just `CI`

Reply:
We can maybe name it just "Scheduled"?

Reply:
Yes, that works